admin/worker/worker.go

```diff
@@ -67,6 +67,9 @@ func (w *Worker) Run(ctx context.Context) error {
 	group.Go(func() error {
 		return w.schedule(ctx, "delete_unused_assets", w.deleteUnusedAssets, 6*time.Hour)
 	})
+	group.Go(func() error {
+		return w.schedule(ctx, "runtime_health_check", w.runtimeHealthCheck, 10*time.Minute)
```

Comment:
Would it be more correct to call it "deployments health check" (`deployments_health_check`) instead?

Reply:
It is both IMO since we also check healths of runtime so pretty subjective TBH. I can change it to `deployments_health_check`.

Reply:
I also prefer `deployments_health_check` and `FindAllocatedDeployments` etc.
Mainly since we might add more components to a deployment in the near future, so the runtime is really only one aspect of the more general deployment health check (but currently the only one).